packages/eas-cli/src/credentials/ios/appstore/distributionCertificate.ts

```diff
@@ -15,51 +15,30 @@ export async function getCertificateBySerialNumberAsync(
   context: RequestContext,
   serialNumber: string
 ): Promise<Certificate> {
-  const cert = (
-    await Certificate.getAsync(context, {
-      query: { filter: { serialNumber: [serialNumber] } },
-    })
-  ).find(item => item.attributes.serialNumber === serialNumber);
+  const cert = (await Certificate.getAsync(context)).find(
+    item => item.attributes.serialNumber === serialNumber
+  );
   if (!cert) {
     throw new Error(`No certificate exists with serial number "${serialNumber}"`);
   }
   return cert;
 }
 
-export async function getMostRecentCertificateAsync(
-  context: RequestContext
-): Promise<Certificate | null> {
-  const [certificate] = await Certificate.getAsync(context, {
-    query: {
-      filter: {
-        certificateType: CertificateType.IOS_DISTRIBUTION,
-      },
-      // Only require a single value.
-      limit: 1,
-      // Sort the serial numbers in reverse order to get the last one
-      // also skip specifying a serial number.
-      sort: '-serialNumber',
-    },
-  });
-  return certificate ?? null;
-}
-
 export async function getDistributionCertificateAync(
   context: RequestContext,
   serialNumber: string
 ): Promise<Certificate | null> {
-  const [certificate] = await Certificate.getAsync(context, {
+  // At most, this returns 2 values.
```

Comment:
```suggestion
  // At most, this returns 3 values.
```
I think the limit is 3 🤔

Reply:
we still query against the type (ios distribution), in which case the max is two.
<img width="313" alt="Screen Shot 2020-12-08 at 12 46 33 PM" src="https://user-images.githubusercontent.com/9664363/101539507-7d2b7180-3953-11eb-8f6c-b7899afeadbf.png">